app/views/planning_application/permitted_development_rights/_constraints.html.erb

```diff
@@ -0,0 +1,19 @@
+<section id="constraints-section">
+  <h2 class="govuk-heading-m">
+    Constraints - including Article 4 direction(s)
+  </h2>
+
+  <% if @planning_application.constraints.empty? %>
+    <p class="govuk-body">
+      There are no constraints
+    </p>
+  <% else %>
+    <ul class="govuk-list govuk-list--bullet">
+      <% @planning_application.constraints.each do |constraint| %>
+        <li><%= constraint %></li>
+      <% end %>
+    </ul>
+  <% end %>
+
+  <%= link_to "Edit constraints", edit_planning_application_constraints_path(@planning_application), class: "govuk-link" %>
```

Comment:
Tiny nit-pick but  I can see from your screenshot that the font for this is smaller than for the 'Edit history' link, maybe because it's because it isn't inside any kind of govuk div. I think you can fix it by making the class `"govuk-link govuk-body"`!